### DISCUSSION TITLE
Entry point for a request

### DISCUSSION THREAD

**Discussion by fz197** [linkerd/linkerd2#8227] [Q&A]
Hello! I'm trying to understand the linkerd proxy source code. which function is the entry point of a request? (A pointer to the codebase will be really helpful!)

Thanks in advance.

**Comment by olix0r** [comment]
It depends on the type of request :)

Generally speaking, though, the [`linkerd-app-oubound`](https://github.com/linkerd/linkerd2-proxy/tree/bf389539ff9a5331deb4fee5fb36dbc92d08933c/linkerd/app/outbound) crate lays out the stacks that handle outbound connections/requests, and [`linkerd-app-inbound`](https://github.com/linkerd/linkerd2-proxy/tree/bf389539ff9a5331deb4fee5fb36dbc92d08933c/linkerd/app/inbound) for inbound traffic.
